interfaces/cython/cantera/test/test_kinetics.py

```diff
@@ -1221,9 +1221,11 @@ def test_modify_invalid(self):
             self.gas.modify_reaction(0, R2)
 
         # different reactants
-        R = self.gas.reaction(7)
+        R = self.gas.reaction(7) # implicitly defined three-body reaction
+        R2 = ct.ElementaryReaction(R.reactants, R.products)
+        R2.rate = R.rate
         with self.assertRaisesRegex(ct.CanteraError, 'Reactants are different'):
-            self.gas.modify_reaction(23, R)
+            self.gas.modify_reaction(23, R2)
```

Comment:
I think the original intent of this test could be satisfied by using any reaction that is still an `ElementaryReaction`. The use of reaction 7 here is arbitrary.